SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -171,10 +171,16 @@ Interest InterestFactory::CreatePartitionInterest(const UAbstractLBStrategy* LBS
 	if (bDebug)
 	{
 		PartitionQuery = Query();
-		PartitionQuery.ResultComponentIds = { SpatialConstants::GDK_DEBUG_TAG_COMPONENT_ID, SpatialConstants::GDK_DEBUG_COMPONENT_ID };
```

Comment:
wouldn't it be better to keep this, and drop the local refresh entity?

Reply:
In terms of trying to reason about what is going on, this way is convoluted. Adding the debug component would make the debug component tag pop into existence. Sure, it works.
But what I really want to have is just a sub view that is able to get authority loss updates.
So I have an easier to implement but farther than what I want to express vs a slightly more verbose way (filters, etc...) closer to what I want to express, and highlighting a lack in sub views implementation (no loopback issue).

Reply:
Discussed offline, we'll keep it as it is as there could be other systems concerned (tombstone) about executing some code and considering the entity as complete in a sub view as soon as they add some component